app/services/transfers/service.go

```diff
@@ -70,54 +71,55 @@ func NewService(
 }
 
 // SanityCheck performs validation on the memo and state proof for the transaction
-func (bs *Service) SanityCheckTransfer(tx mirror_node.Transaction) (*memo.Memo, error) {
+func (ts *Service) SanityCheckTransfer(tx mirror_node.Transaction) (*memo.Memo, error) {
 	m, e := memo.FromBase64String(tx.MemoBase64)
 	if e != nil {
 		return nil, errors.New(fmt.Sprintf("Could not parse transaction memo. Error: [%s]", e))
 	}
 
-	stateProof, e := bs.mirrorNode.GetStateProof(tx.TransactionID)
-	if e != nil {
-		return nil, errors.New(fmt.Sprintf("Could not GET state proof. Error [%s]", e))
-	}
-
-	verified, e := proof.Verify(tx.TransactionID, stateProof)
-	if e != nil {
-		return nil, errors.New(fmt.Sprintf("State proof verification failed. Error [%s]", e))
-	}
-
-	if !verified {
-		return nil, errors.New("State proof not valid")
-	}
+	// TODO: Uncomment when State Proof Method gets updated accordingly
```

Comment:
Is there a problem with the state proof?

Reply:
The structure retrieved from the GET query is changed entirely. It needs update.